test/performance/README.md

```diff
@@ -0,0 +1,25 @@
+# Performance Tests
+
+To run the performance tests, execute the following from the root of the repository:
+
+```
+$ kind create cluster
+$ bash hack/quick-install.sh
+$ bash test/performance/run.sh
+<.. omitted output ..>
+RESULTS in requests per second (req/s):
+get-empty: 733 req/s
+create: 97 req/s
+get1: 392 req/s
+get: 7 req/s
+get1000: 1 req/s
+```
```

Comment:
Suggestion for making markdown readers exposing the copy button properly:

```suggestion
```bash
kind create cluster
bash hack/quick-install.sh
bash test/performance/run.sh
```

I am also removing the output part because I didn't see it when I ran the tests.
BTW, I find useful to have explicitly somewhere that the results found under `perf-results` are shown in milliseconds.

Reply:
I wrote something about it, let me know if it is enough.

Reply:
The `README.md` looks nice with the new information :) 

But can you please add the `bash` word after initializing the code block? That makes markdown readers nicely highlight the syntax in it

Reply:
Also, could you add as a note, the minimum version of python expected, or at least what version is used in our CI? I ran it with 3.12 but the github action is ran on `ubuntu-latest` and my guess is that its python version isn't the latest one